src/query/injector.ts

```diff
@@ -49,7 +49,7 @@ export class QueryInjector {
     let child: InjectedQuery;
 
     try {
-      const query = parseQuery(pendingQuery.source);
+      const query = parseQuery(pendingQuery.source, this.app.workspace.getActiveFile().basename);
```

Comment:
I'm not sure this will always be correct - really we want the file where the block we are 'replacing' lives. I'm not sure off the top of my head if that's available through the markdown post-processor API that Obsidian offers.

Reply:
Good point, I was worried about that too. It looks like the context parameter of the post-process API gives this info. I can revise to use that with some additional regex to grab the name.

For reference: https://docs.obsidian.md/Reference/TypeScript+API/MarkdownPostProcessorContext/MarkdownPostProcessorContext

Relevant snippet to test: 
```
class QueryInjector {
    constructor(app) {
        this.app = app;
        this.pendingQueries = [];
    }
    onNewBlock(source, el, ctx) {
        const pendingQuery = {
            source: source,
            target: el,
            ctx: ctx,
        };
        if (typeof this.api == "undefined") {
            this.pendingQueries.push(pendingQuery);
            return;
        }
	console.log("context.sourcePath:" + ctx.sourcePath);
        this.injectQuery(pendingQuery);
    }
```
Returns: `context.sourcePath:path/to/noteFile.md`